server/election.py

```diff
@@ -1,36 +1,149 @@
-import uuid
-from flask import jsonify, request
+import uuid, hashlib, json
+from flask import jsonify, request, make_response
 
-from . import app
+from . import app, API_URL_PREFIX
 from .models import *
+from .util.jsonschema import validate
+from .schemas import ELECTION_SCHEMA
+from .security import with_mailelection_admin
 
 
-def serialize_election(election):
-    return {
+def serialize_election(election, expand=False):
```

Comment:
Should we rename this file to `mailelection` and all `election` function params to `mailelection`? I would like to avoid future confusion between data objects.

Reply:
good call